wallet-new/integration/Functions.hs

```diff
@@ -10,7 +10,7 @@ import           Data.Coerce (coerce)
 import           Data.List (delete)
 import           Data.List.NonEmpty (fromList)
 
-import           Control.Lens ((+~), (<>~), (?~), each, filtered, at)
+import           Control.Lens (at, each, filtered, (+~), (<>~), (?~))
```

Comment:
Sorry for the lame question, but have you used `stylish-haskell` (as per the style-guide) to format these imports? (in other terms, I'm trying to understand why this changed 😁 )

Reply:
Yes, I use it.

Could be a difference in stylish-haskell version or what config it's picking up?

Reply:
Possibly. Even more confusingly, the `.stylish-haskell.yml` file changed between `master` and `develop`, with the one used in the latter needing the latest version of `stylish-haskell` to work (> 0.8.2 IIRC)